package.json

```diff
@@ -37,18 +37,18 @@
     "elliptic-curve-solidity": "0.2.2"
   },
   "devDependencies": {
-    "eslint": "^6.8.0",
-    "eslint-config-standard": "^14.1.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint": "^7.13.0",
+    "eslint-config-standard": "^16.0.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "eth-gas-reporter": "^0.2.15",
+    "eslint-plugin-standard": "^4.1.0",
+    "eth-gas-reporter": "0.2.18",
     "ethlint": "^1.2.5",
-    "ganache-cli": "^6.9.1",
-    "solidity-coverage": "^0.7.2",
+    "ganache-cli": "^6.12.1",
+    "solidity-coverage": "^0.7.12",
     "solium": "^1.2.5",
-    "truffle": "^5.1.17",
+    "truffle": "^5.1.53",
```

Comment:
Here the discussion is if we should avoid using `^` because it is also prone to future errors.